src/modules/ROOT/pages/changelog/202303/01.json

```diff
@@ -0,0 +1,889 @@
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/760/commits/954419bac6c3d19e3c5c46adf93d908528ad18df"
+      ]
+    },
+    {
+      "what": [
+        "ability to serve data as response in an integration gateway route.",
+        "Also Route description field, Filters sorting, Profiler hits purging and Integration gateway settings were added."
+      ],
+      "why": [
+        "to give an endpoint response greater extensibility to integration gateway as different data can be returned."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/775/commits/b250b9329fc05bc6da2f65fb8632b3b2bda937dc",
+        "https://github.com/cortezaproject/corteza/pull/775/commits/921d4988c75159bc25a6296255327d8c99c762f8",
+        "https://github.com/cortezaproject/corteza/pull/775/commits/a64f4c18b3d428bb207ab9f7f95c5a42324a9bab",
+        "https://github.com/cortezaproject/corteza/pull/775/commits/352865b4fb9aab5d0a1697bca88987f06d4e913e",
+        "https://github.com/cortezaproject/corteza/pull/775/commits/6c423084b336d35367c733457d41a138f2d2d5c0",
+        "https://github.com/cortezaproject/corteza/pull/775/commits/31ca5d6c297ec1569f01d72a9389a5590e3486b9",
+        "https://github.com/cortezaproject/corteza/pull/775/commits/11dccaea9f299895e5637929c02e1b0dfb518bcd"
+      ]
+    },
+    {
+      "what": [
+        "a configuration option to enable delete-record listing.",
+        "Support for viewing and undeleting a deleted record and a support for undeleting multiple deleted records was also added."
+      ],
+      "why": [
+        "to allow users to view and restore deleted records."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/740/commits/101c1c7e3026285052c9a731684c2cb278047144"
+      ]
+    },
+    {
+      "what": [
+        "a configuration option on charts to disable or enable animation."
+      ],
+      "why": [
+        "to allow users more chart configuration options."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/806/commits/f9cc910e5a769538c83fa96acb53f04af0f9efe8"
+      ]
+    },
+    {
+      "what": [
+        "BETWEEN and NOT BETWEEN operator support for single value fields(number and date-time record fields)."
+      ],
+      "why": [
+        "to extend record filtering support for range filtering of numeric and date-time for single value field."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/813/commits/2027409b0d8875a48fd0017264ba03f5535733ec"
+      ]
+    },
+    {
+      "what": [
+        "new support for a fully customizable navigation page block with multiple navigation item type and customizable options for the navigation list items."
+      ],
+      "why": [
+        "to extend the possibilities of offered blocks to the users and to have the ability to show one or two level of dynamic and static navigation."
```

Comment:
`to extend the possibilities of offered blocks to the users`

I'd omit this bit; sounds redundant